pool/app/pool_database/pool_database.ml

```diff
@@ -12,3 +12,14 @@ module Logger = struct
     ;;
   end
 end
+
+let test_and_create url label =
+  let%lwt connection =
+    url
+    |> Uri.of_string
+    |> CCFun.flip Caqti_lwt.with_connection (fun _ -> Lwt_result.return ())
```

Comment:
I used `with_connection`, because it will close the connection once it succeeds or fails: https://paurkedal.github.io/ocaml-caqti/caqti-lwt/Caqti_lwt/index.html#val-with_connection